form.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css" />
+  <script src="form.js" defer></script>
+  <title>Form</title>
+</head>
+<body>
+  <body data-js="body">
+    <header class="header">
+    <h1
+      class="header__headline"
+      >Your Q&A's
+    </h1>
```

Comment:
your Prettier appears not to work properly for this part of the code.
Does it format your code properly when you save a file?

Reply:
Yes, it usually does...but my site is kind of special...for example:
I've bild in two body_opening_tags...and in think there are some more Mistakes i've made...🤣
Thank you very much for taking a look 🙏🏼 I'll try to fix everything.😇👍🏼